layouts/shortcodes/lang_instrumentation_index_head.md

```diff
@@ -1,7 +1,18 @@
 {{ $data := index $.Site.Data.instrumentation.languages (.Get 0) }}
 {{ $name := $data.name }}
 {{ $relUrl := printf "https://github.com/open-telemetry/opentelemetry-%s/releases" (.Get 0) -}}
-
+{{ $tracesStatus := $data.status.traces | humanize }}
+{{ $metricsStatus := $data.status.metrics | humanize }}
+{{ $logsStatus := $data.status.logs | humanize }}
+{{ if in "Stable Experimental" $tracesStatus }}
+    {{ $tracesStatus = printf "[%s](/docs/reference/specification/versioning-and-stability/#%s)" $tracesStatus $data.status.traces }}
+{{ end }}
+{{ if in "Stable Experimental" $metricsStatus }}
+    {{ $metricsStatus = printf "[%s](/docs/reference/specification/versioning-and-stability/#%s)" $metricsStatus $data.status.metrics }}
+{{ end }}
+{{ if in "Stable Experimental" $logsStatus }}
```

Comment:
Should "Development" also be added?

Reply:
To what URL should it point then?

https://opentelemetry.io/docs/reference/specification/versioning-and-stability/

knows Stable, Experimental, Deprecated, Removed

Reply:
Not sure, but this is the status for JS + logs, for example.

Reply:
yeah, there are some terms used which are not officially supported, that's why I check for Stable & Experimental and skip the rest.